base_import_security_group/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Could you add the target tag as it is in the [README template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)?